pkg/client/client.go

```diff
@@ -759,49 +768,28 @@ func (c *Client) getProviderConfig(providerName string) (*config.RequiredProvide
 	return providerConfig, nil
 }
 
-func FilterPolicies(args []string, configPolicies []*config.Policy, policyName, subPath string) ([]*config.Policy, error) {
-	var policies []*config.Policy
-
-	if len(args) > 0 {
-		remotePolicy, err := policy.ParsePolicyFromArgs(args)
-		if err != nil {
-			return nil, err
-		}
-		policyConfig, err := remotePolicy.ToPolicyConfig()
-		policyConfig.SubPath = subPath
-		if err != nil {
-			return nil, err
-		}
-		policies = append(policies, policyConfig)
-	} else {
-		policies = configPolicies
+func (c *Client) setupTableCreator(ctx context.Context) error {
+	if c.TableCreator != nil {
+		c.Logger.Debug("table creator already set")
```

Comment:
Warning level instead? Shouldn't happen anyway

Reply:
it can happen if set as a functional option, so maybe keep it as debug.